Formula/wilson.rb

```diff
@@ -0,0 +1,15 @@
+class Wilson < Formula
+  desc "Wilson - developer's routine tasks automation toolkit. Simple modern alternative to GNU Make 🧰"
+  homepage "https://github.com/trntv/wilson"
+  url "https://github.com/trntv/wilson/releases/download/0.3.1/wilson-darwin-amd64.tar.gz"
+  sha256 "5e428579afbc4005b3730d34f3166f8b27c738bade2e95c49ec465120c0c307a"
+  version "0.3.1"
+
+  def install
+    bin.install "wilson_darwin_amd64" => "wilson"
+  end
+
+  test do
+    system "#{bin}/wilson", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.